VTEX - Catalog API Seller Portal.json

```diff
@@ -492,6 +530,11 @@
                                         "description": "Product's brand unique identifier number.",
                                         "example": "1"
                                     },
+                                    "description":{
+                                        "type":"string",
+                                        "description": "New description data for Camiseta VTEX 10.",
```

Comment:
Não colocaria o nome do exemplo aqui na descrição, colocaria o que ele significa, como `New description data for SKU/product`